### PR TITLE
Fix bug when run command `celery`

### PR DESCRIPTION
**Pull request by ThongLe**
There's a bug when run `python manage celery` related to attribute `preload_options` not existed in `CeleryCommand`.

### CODE REVIEW

File: djcelery/management/commands/celery.py [hunk 1]
```diff
@@ -17,7 +17,6 @@ class Command(CeleryCommand):
         preload_options = base.preload_options if base.preload_options else []
     else:
         preload_options = []
-    preload_options = base.preload_options if base.preload_options else []
```
Comment (liquidpele) [inline djcelery/management/commands/celery.py]: It looks like this and the if/else block could be replace with just:

    preload_options = getattr(base, 'preload_options', []) or []

Comment (ThongLe) [reply djcelery/management/commands/celery.py]: I fixed it. Thanks, @liquidpele :).

**Comment by ThongLe** [comment]
@auvipy , it's failed looked like some others. I think it should be fixed somewhere but this PR.

**Comment by Peng-Xu** [comment]
'CeleryCommand' object has no attribute 'preload_options'

**Comment by liquidpele** [comment]
flake8 tests are failing now.

File: djcelery/management/base.py [hunk 2]
```diff
@@ -35,15 +35,20 @@ def _init(self, *args, **kwargs):
...
 
+            def _validate_thread(self):
+                is_valid = self.allow_thread_sharing
+                is_valid = is_valid or self._thread_ident == _get_ident()
+                return is_valid
+
             def _validate_thread_sharing(self):
-                if (not self.allow_thread_sharing and
-                        self._thread_ident != _get_ident()):
+                if not self.validate_thread():
```
Comment (liquidpele) [inline djcelery/management/base.py]: I think you are missing the underscore for the method name you added?

File: djcelery/management/base.py [hunk 3]
```diff
@@ -35,15 +35,20 @@ def _init(self, *args, **kwargs):
...
                     raise DatabaseError(
                         DB_SHARED_THREAD % (
                             self.alias, self._thread_ident, _get_ident()),
                     )
 
             BaseDatabaseWrapper.__init__ = _init
+            BaseDatabaseWrapper.validate_thread = _validate_thread
```
Comment (ThongLe) [inline djcelery/management/base.py]: @liquidpele , it doesn't have underscore here.

Comment (liquidpele) [reply djcelery/management/base.py]: Oh, I see.  Didn't expand to look at the whole file.  Thx.

**Pull request merged**